packages/rancher-logging/rancher-logging.patch

```diff
@@ -98,13 +98,13 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-logging/charts-original/values
  
  affinity: {}
  
-@@ -76,4 +81,49 @@
+@@ -76,4 +81,46 @@
  monitoring:
    # Create a Prometheus Operator ServiceMonitor object
    serviceMonitor:
 -    enabled: false
 \ No newline at end of file
-+    enabled: true
++    enabled: false
```

Comment:
It looks like this wasn't upstream change, whats reasoning for this ?

Reply:
I saw `serviceMonitor` getting installed with our 3.7.0 chart, and failing install. @paynejacob recommended that we disable its installation.

Reply:
consensus is to disable it